charts/terraform-pod/templates/deployment.yaml

```diff
@@ -0,0 +1,165 @@
+              name: scripts
+            - mountPath: /home/terraform/git-crypt-key
+              name: git-crypt-key
+              subPath: git-crypt-key
+              readOnly: true
+            - name: github-token
+              mountPath: /home/terraform/.github/token
+              subPath: token
+      containers:
+        - name: terraform
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - /bin/bash
+            - -c
+            - |
+              source ../scripts/github.sh
+              source ../scripts/slack.sh
+              {{ .Values.terraform.initCommand }}
+              slack_notificaton_ready
+
+              printf %140s |tr " " "="
+              echo ""
+              echo "kubectl exec -ti \$(kgpo -l app.kubernetes.io/name=terraform-pod --no-headers -o="custom-columns=NAME:.metadata.name") -c terraform -- bash"
+              echo "Please run [make _env apply] once you get in the pod"
+              echo "healthy" > /tmp/healthy
+              printf %140s |tr " " "="
+              sleep infinity
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          env:
+            {{ include "terraform-pod.default-env-vars" . | nindent 12 }}
+            {{ include "terraform-pod.terraform-env-vars" . | nindent 12 }}
+            {{ include "terraform-pod.aws-env-vars" . | nindent 12 }}
+            {{- if .Values.infracost.enabled }}
+            {{ include "terraform-pod.infracost-env-vars" . | nindent 12 }}
+            {{- end }}
+            {{- with .Values.extraEnvVars }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+          resources:
+            {{ toYaml .Values.resources | nindent 12 }}
+          readinessProbe:
+            exec:
+              command:
+              - cat
+              - /tmp/healthy
+            initialDelaySeconds: 30
+            periodSeconds: 5
+          volumeMounts:
+            - name: workspace
+              mountPath: /home/terraform/workspace
+            - name: scripts
+              mountPath: /home/terraform/scripts
+            - name: github-terraform-cicd-app-private-key
+              mountPath: /keys
+            - name: saritasa-terraform-modules-deploy-key
+              mountPath: /home/terraform/.ssh/id_rsa
+              subPath: ssh-privatekey
+            - name: github-known-hosts
+              mountPath: /home/terraform/.ssh/known_hosts
+              subPath: config.ssh
+            - name: github-token
+              mountPath: /home/terraform/.github/token
+              subPath: token
+            {{- with .Values.extraVolumeMounts }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+
+      {{- with .Values.nodeSelector }}
```

Comment:
Please let's use more compact way, like you do above (I think `with` construction is redundant when we use this variable just once):

```
imagePullSecrets:
          {{- toYaml .Values.imagePullSecrets | nindent 8 }}
```

Let's update it everywhere? Manifest will have less lines and would be easier to read

Reply:
Explained above.